.github/workflows/ci.yml

```diff
@@ -302,6 +308,24 @@ jobs:
       - name: Run cargo build
         run: ./scripts/test_no_std ${{ env.RUST_NIGHTLY_TOOLCHAIN }}
 
+  unstable:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - uses: actions-rs/toolchain@v1.0.7
+        id: toolchain
+        with:
+          toolchain: ${{ needs.env.outputs.msrv }}
+
+      - uses: camshaft/rust-cache@v1
+
+      - name: Test s2n_quic --cfg unstable
+        run: ./scripts/test_unstable
```

Comment:
I think we should just test unstable features without having to opt into it; makes dev a lot nicer. So do `cfg(any(test, all(s2n_quic_unstable, feature = "")))`.

Reply:
` RUSTFLAGS="--cfg s2n_internal_ci" RUSTDOCFLAGS="--cfg s2n_internal_ci" cargo +stable clippy --all-features --all-targets -- -D warnings`.. the RUSTFLAGS, esplecialy the `internal_use` are starting to bleed into regular usage which is not ideal.